src/main/java/de/tum/in/www1/artemis/repository/plagiarism/PlagiarismCaseRepository.java

```diff
@@ -19,16 +19,18 @@
 public interface PlagiarismCaseRepository extends JpaRepository<PlagiarismCase, Long> {
 
     @Query("""
-            SELECT DISTINCT plagiarismCase FROM PlagiarismCase plagiarismCase
+            SELECT DISTINCT plagiarismCase
+            FROM PlagiarismCase plagiarismCase
             LEFT JOIN FETCH plagiarismCase.post
             LEFT JOIN FETCH plagiarismCase.plagiarismSubmissions plagiarismSubmission
             WHERE plagiarismCase.student.login = :studentLogin
```

Comment:
```suggestion
            FROM PlagiarismCase plagiarismCase
                LEFT JOIN FETCH plagiarismCase.post
                LEFT JOIN FETCH plagiarismCase.plagiarismSubmissions plagiarismSubmission
            WHERE plagiarismCase.student.login = :studentLogin
```

Also indent the joins.